src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -0,0 +1,211 @@
+                    shouldShowBackButton
+                    onCloseButtonPress={() => Navigation.dismissModal()}
+                    onBackButtonPress={() => Navigation.goBack()}
+                />
+                <Form
+                    style={[styles.flexGrow1, styles.ph5]}
+                    formID={ONYXKEYS.FORMS.LEGAL_NAME_FORM}
+                    validate={this.validate}
+                    onSubmit={this.onSubmit}
+                    submitButtonText={this.props.translate('common.save')}
+                    enabledWhenOffline
+                    footerContent={
+                        (
+                            <Pressable
+                                onPress={this.openPrivacyURL}
+                                accessibilityRole="link"
+                                href={CONST.PRIVACY_URL}
+                                style={[styles.mh5, styles.mb3, styles.alignSelfStart]}
+                            >
+                                <View style={[styles.flexRow]}>
+                                    <Text style={[styles.mr1, styles.label, styles.link]}>
+                                        {this.props.translate('common.privacy')}
+                                    </Text>
+                                </View>
+                            </Pressable>
+                        )
+                    }
+                >
+                    <View style={[styles.mv4, styles.justifyContentCenter, styles.alignItemsCenter]}>
+                        <MultipleAvatars
+                            size={CONST.AVATAR_SIZE.LARGE}
+                            icons={this.getAvatars()}
+                            shouldStackHorizontally
+                            secondAvatarStyle={[
+                                styles.secondAvatarInline,
+                            ]}
+                            avatarTooltips={this.getAvatarTooltips()}
+                        />
+                    </View>
+                    <View style={[styles.mb5]}>
+                        <Text>
+                            {this.props.translate('workspace.inviteMessage.inviteMessagePrompt')}
+                        </Text>
+                    </View>
+                    <View style={[styles.mb3]}>
+                        <TextInput
+                            label={this.props.translate('workspace.inviteMessage.personalMessagePrompt')}
+                            autoCompleteType="off"
+                            autoCorrect={false}
+                            numberOfLines={4}
+                            textAlignVertical="top"
+                            multiline
+                            containerStyles={[styles.workspaceInviteWelcome]}
+                            defaultValue={this.state.welcomeNote}
+                            value={this.state.welcomeNote}
+                            onChangeText={text => this.setState({welcomeNote: text})}
```

Comment:
@abdulrahuman5196 I understand that the `welcomeNote` changes when the locale changes, but the Form will not be valid if we manage the value in the local component. We can set a new draft value for the form whenever the locale is changed, and we don't need `value` nor `onChangeText` in this input.